runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -2534,6 +2542,17 @@ private static String longBossName(String boss)
 			case "fortis colosseum":
 				return "Sol Heredit";
 
+			case "bird egg":
+			case "bird eggs":
+			case "bird's egg":
+			case "bird's eggs":
+			case "egg":
+			case "eggs":
+			case "evil chicken":
```

Comment:
The last 5 cases here ("egg" onwards, in case I don't have them all selected because I'm looking at this on mobile) feel either too generic or in the case of "evil chicken", may be confused for something else, being the Evil Chicken from Recipe for Disaster.